drivers/clock_control/nrf5_power_clock.c

```diff
@@ -344,8 +345,12 @@ static void _power_clock_isr(void *arg)
 			 */
 			NRF_CLOCK->INTENCLR = CLOCK_INTENCLR_LFCLKSTARTED_Msk;
 
-			/* Start HF Clock */
-			ctto = 1;
+			/* Start HF Clock if LF RC is used. */
+			if (CLOCK_LFCLKSRCCOPY_SRC_RC ==
+			(NRF_CLOCK->LFCLKSRCCOPY & CLOCK_LFCLKSRCCOPY_SRC_Msk)
+			) {
```

Comment:
At least in Linux it is ok to break the 80 character rule when the alternative is hard-to-read code or violating some other coding style (like your `) {` on a separate line). So I'd do the split after == but then put the rest properly on the same line, even though it'll create a coding style warning (we can override that when merging).

Reply:
ok

Reply:
done